examples/token-escrow/programs/token-escrow/src/escrow_with_compressed_pda/withdrawal.rs

```diff
@@ -140,45 +144,12 @@ fn cpi_compressed_pda_withdrawal<'info>(
         new_address_params: Vec::new(),
         compress_or_decompress_lamports: None,
         is_compress: false,
-        signer_seeds: seeds.iter().map(|seed| seed.to_vec()).collect(),
+        signer_seeds: signer_seeds.iter().map(|seed| seed.to_vec()).collect(),
         cpi_context: Some(cpi_context),
     };
 
-    let mut inputs = Vec::new();
-    InstructionDataInvokeCpi::serialize(&inputs_struct, &mut inputs).unwrap();
-
-    let cpi_context_account = match Some(cpi_context) {
-        Some(cpi_context) => Some(
-            ctx.remaining_accounts
-                .get(cpi_context.cpi_context_account_index as usize)
-                .unwrap()
-                .to_account_info(),
-        ),
-        None => return err!(EscrowError::CpiContextAccountIndexNotFound),
-    };
-
-    let cpi_accounts = light_system_program::cpi::accounts::InvokeCpiInstruction {
-        fee_payer: ctx.accounts.signer.to_account_info(),
-        authority: ctx.accounts.token_owner_pda.to_account_info(),
-        registered_program_pda: ctx.accounts.registered_program_pda.to_account_info(),
-        noop_program: ctx.accounts.noop_program.to_account_info(),
-        account_compression_authority: ctx.accounts.account_compression_authority.to_account_info(),
-        account_compression_program: ctx.accounts.account_compression_program.to_account_info(),
-        invoking_program: ctx.accounts.self_program.to_account_info(),
-        sol_pool_pda: None,
-        decompression_recipient: None,
-        system_program: ctx.accounts.system_program.to_account_info(),
-        cpi_context_account,
-    };
-    let signer_seeds: [&[&[u8]]; 1] = [&seeds[..]];
-    let mut cpi_ctx = CpiContext::new_with_signer(
-        ctx.accounts.light_system_program.to_account_info(),
-        cpi_accounts,
-        &signer_seeds,
-    );
-    cpi_ctx.remaining_accounts = ctx.remaining_accounts.to_vec();
+    exec_verify(ctx, &inputs_struct, &[&signer_seeds])?;
```

Comment:
naming choice is not immediately obvious to me.
What about one of the following?
1. invoke_light_system_program
2. invoke_system_program
3. verify_compressed_transaction
4. execute_compressed_transaction
5. invoke_with_signer
6. invoke_from_program
7. invoke_program
8. invoke_cpi

5 is my favorite because the import path makes it clear that it is the light system program and it is the same lingo as the solana sdk.
6 and 7 are even more explicit though.
8 would be consist with system program naming.

Reply:
of the above `invoke_cpi` fits well with the underlying cpi instruction, agreed. thing here is people need to call this every time they want to write to any state / to verify and apply their state transition. hence the intuition to provide a dedicated very high-level naming for this wrapper.
* `light_sdk::verify()` (pref, because shorter)
* `light_sdk::verify_state_transition()`
i like these bc they're most descriptive at a very high level

Reply:
renamed to verify for now - let's revisit this